### PR TITLE
WIP: broadcastTx

### PR DESCRIPTION
**Pull request by matiu**
broadcast TX. Test still missing.

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -502,7 +513,7 @@ CopayServer.prototype.signTx = function(opts, cb) {
         if (err) return cb(err);
 
         if (txp.status == 'accepted') {
-          self._broadcastTx(txp.rawTx, function(err, txid) {
+          self._broadcastTx(txp, wallet.getNetworkName(), function(err, txid) {
```
Comment (isocolsky) [inline lib/server.js]: I'm thinking maybe we should include the network inside the txp

**Comment by matiu** [comment]
Updated: tests added and comment addressed.

File: lib/server.js [hunk 2]
```diff
@@ -273,8 +280,9 @@ CopayServer.prototype._getUtxos = function(opts, cb) {
 
     var addressStrs = _.pluck(addresses, 'address');
     var addressToPath = _.indexBy(addresses, 'address'); // TODO : check performance
+    var networkName = Bitcore.Address(addressStrs[0]).toObject().networkName;
```
Comment (isocolsky) [inline lib/server.js]: nice :)

File: lib/server.js [hunk 3]
```diff
@@ -502,10 +511,10 @@ CopayServer.prototype.signTx = function(opts, cb) {
         if (err) return cb(err);
 
         if (txp.status == 'accepted') {
-          self._broadcastTx(txp.rawTx, function(err, txid) {
-            if (err) return cb(err);
+          self._broadcastTx(txp, wallet.getNetworkName(), function(err, txid) {
+            if (err) return cb(err, txp);
```
Comment (isocolsky) [inline lib/server.js]: should be ```if (err) return cb(err)``` ?

**Comment by matiu** [comment]
updated.

**Pull request merged**